pynecone/pc.py

```diff
@@ -159,10 +163,17 @@ def export():
     # Compile the app in production mode and export it.
     utils.console.rule("[bold]Compiling production app and preparing for export.")
     app = utils.get_app().app
-    utils.export_app(app, zip=True)
-    utils.console.rule(
-        "Backend & Frontend compiled. See [green bold]backend.zip[/green bold] and [green bold]frontend.zip[/green bold]."
-    )
+    utils.export_app(app, zip=zipping)
+    if zipping:
```

Comment:
Now that you've added a way to do the zipping in Windows, should we remove this flag and just always do it?

Reply:
Maybe someone want to export it without zipping, so we can leave the parameter in. What do you think about it?